src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,39 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  selectedTodo: Todo;
+  setSelectedTodo: React.Dispatch<React.SetStateAction<Todo | null>>;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  setSelectedTodo,
+}) => {
+  const [user, setUser] = useState<User | null>();
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [loading, setLoading] = useState(true);
```
to remove modal blinks when first time try to open